generators/ionic/resources/base/README.md

```diff
@@ -0,0 +1,159 @@
+# The Ionic JHipster Starter 🤓
```

Comment:
Will this file be included as the README in the new project? If so, we should adjust it and add JHipster commands to it.

Reply:
Yes it will